dissect/target/plugins/apps/webserver/iis.py

```diff
@@ -84,11 +93,23 @@ def get_log_dirs(self) -> list[tuple[str, Path]]:
         except (ElementTree.ParseError, DissectFileNotFoundError) as e:
             self.target.log.warning(f"Error while parsing {self.config}: {e}")
 
+        for log_path in self.DEFAULT_LOG_PATHS:
+            for log_file in self.target.fs.glob(log_path):
+                try:
+                    log_dir = self.target.fs.path(log_file).parents[1]
```

Comment:
Since this is the only place you use `DEFAULT_LOG_PATHS`, I don't really get why you need to get `parents[1]`? Why not just specify two dirs "higher" for each path? 

Or do you use it to check if there are any files in that directory at all? Maybe then it would be nice to add a small comment saying so.

Reply:
This is because we use the acquire glob patterns, which may contain wildcards at arbitrary path segments "a/*/b" "a/b/c/*/*/"  etc. So, to be able to reuse the, we basically glob the files and then inquire about the parent folder. Otherwise you would have to create several recursive loops, or something like that.

Reply:
But that's only for the last item in the list, the others are all paths without a glob (if you see it from parents[1], which is the `LogFiles` directory). In my opinion, it now iterates those paths unnecessarily. 

Maybe check for `*` in the path entry and then do the logic below. Else just check if the path exists and add it?